src/go/rpk/pkg/cli/cmd/cluster/storage/recovery/start.go

```diff
@@ -0,0 +1,110 @@
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func newStartCommand(fs afero.Fs) *cobra.Command {
+	var (
+		topicNamePattern string
+		wait             bool
+		pollingInterval  time.Duration
+	)
+
+	cmd := &cobra.Command{
+		Use:   "start",
+		Short: "Start the recovery process",
+		Long: `Start the recovery process.
+		
+This command starts the process of restoring topics from the archival bucket.
+If the wait flag (--wait/-w) is set, the command will poll the status of the
+recovery process until it's finished.`,
+		Run: func(cmd *cobra.Command, args []string) {
+			p := config.ParamsFromCommand(cmd)
+			cfg, err := p.Load(fs)
+			out.MaybeDie(err, "unable to load config: %v", err)
+
+			client, err := admin.NewClient(fs, cfg)
+			out.MaybeDie(err, "unable to initialize admin client: %v", err)
+
+			ctx := cmd.Context()
+
+			_, err = client.StartAutomatedRecovery(ctx, topicNamePattern)
+			var he *admin.HTTPResponseError
+			if errors.As(err, &he) {
+				if he.Response.StatusCode == 404 {
+					body, bodyErr := he.DecodeGenericErrorBody()
+					if bodyErr == nil {
+						out.Die("Not found: %s", body.Message)
+					}
+				} else if he.Response.StatusCode == 400 {
+					body, bodyErr := he.DecodeGenericErrorBody()
+					if bodyErr == nil {
+						out.Die("Cannot start auto-restore: %s", body.Message)
+					}
+				}
+			}
+
+			out.MaybeDie(err, "error starting auto-restore: %v", err)
+			fmt.Println("Successfully started auto-restore")
+
+			if !wait {
+				return
+			}
+
+			fmt.Println("Waiting for auto-restore to complete...")
+
+			for {
+				status, err := client.PollAutomatedRecoveryStatus(ctx)
+				out.MaybeDie(err, "failed to poll automated recovery status: %v", err)
+
+				pending := false
+				for _, topicDownload := range status.TopicDownloads {
+					if topicDownload.PendingDownloads > 0 {
+						pending = true
+						break
+					}
+				}
+
+				if !pending {
+					failedPartitionReplicas := []string{}
+					for _, topicDownload := range status.TopicDownloads {
+						if topicDownload.FailedDownloads > 0 {
+							failedPartitionReplicas = append(failedPartitionReplicas, topicDownload.TopicNamespace)
+						}
+					}
+
+					if len(failedPartitionReplicas) > 0 {
+						out.Die("automated recovery failed to download partition replicas: %v", failedPartitionReplicas)
+					}
+
+					break
+				}
+				time.Sleep(pollingInterval)
+			}
+		},
+	}
+
+	cmd.Flags().StringVar(&topicNamePattern, "topic-name-pattern", ".*", "A regex pattern to match topic names against. Only topics whose names match this pattern will be restored. If not passed, all topics will be restored.")
+	cmd.Flags().BoolVarP(&wait, "wait", "w", false, "Wait until auto-restore is complete.")
+	cmd.Flags().DurationVar(&pollingInterval, "polling-interval", 5*time.Second, "The interval in seconds between each status check. Ignored if --wait is not passed.")
```

Comment:
I might be wrong, but doesn't the durationVar allow passing any duration string? (not just seconds): 1m, 1h are valid intervals.